ibridges/session.py

```diff
@@ -293,6 +295,8 @@ def _translate_irods_error(exc) -> Exception:  # pylint: disable=too-many-return
             return LoginError("Host, port, irods_client_server_policy or "
                               "irods_client_server_negotiation not set correctly in "
                               "irods_environment.json")
+    if isinstance(exc, TypeError):
```

Comment:
This is kind of horrible, but I suppose there's not much we can do on our side here?

Reply:
The TypeError changes its message depending on which key in the iRODS file is missing. So I could not think of a more elegant way to get this message passed on :(